account_credit_control_dunning_fees/readme/CONTRIBUTORS.rst

```diff
@@ -2,3 +2,6 @@
 * Guewen Baconnier <guewen.baconnier@camptocamp.com>
 * Adrien Peiffer (acsone)
 * Akim Juillerat <akim.juillerat@camptocamp.com>
+* `Trobz <https://trobz.com>`_:
+
+    * Nguyễn Minh Chiến <chien@trobz.com>
```

Comment:
Please add your name as previous contributors did. Also, the name of your company is already on your email domain.